File: h2o-docs/src/booklets/v2_2015/source/GLM_Vignette_code_examples/glm_negativebinomial_example.R
library(h2o)
h2o.init()
h2o_df = h2o.importFile("http://h2o-public-test-data.s3.amazonaws.com/smalldata/glm_test/Motor_insurance_sweden.txt")
predictors <- c["Payment", "Insured", "Kilometres", "Zone", "Bonus", "Make"]
response <- "Claims"
negativebinomial.fit <- h2o.glm(x=predictors, y=response, training_frame=h2o_df, family="negativebinomial", link="identity", theta=0.5)

File: h2o-docs/src/booklets/v2_2015/source/GLM_Vignette_code_examples/glm_negativebinomial_example.py
import h2o
from h2o.estimators.glm import H2OGeneralizedLinearEstimator
h2o.init()
h2o_df = h2o.h2o.import_file("http://h2o-public-test-data.s3.amazonaws.com/smalldata/glm_test/Motor_insurance_sweden.txt")
predictors = ["Payment", "Insured", "Kilometres", "Zone", "Bonus", "Make"]
response = "Claims"
negativebinomial_fit = H2OGeneralizedLinearEstimator(family="negativebinomial", link="identity", theta=0.5)
negativebinomial_fit.train(x=predictors, y=response, training_frame=h2o_df)
